benchmarks/instruction_counts/main.py

```diff
@@ -24,4 +28,13 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    modes = {
+        "debug": main,
+        "ci": ci.main,
+    }
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--mode", type=str, choices=list(modes.keys()), default="debug")
+
+    args, remaining_args = parser.parse_known_args(sys.argv)
+    modes[args.mode](remaining_args[1:])
```

Comment:
What is the number of threads argument for the CI?

Reply:
The benchmark controls num_threads on a per benchmark basis rather than a global basis.